lib/plug/conn/status.ex

```diff
@@ -132,14 +132,15 @@ defmodule Plug.Conn.Status do
   end
 
   def reason_phrase(code) do
-    raise ArgumentError, "Unknown status code #{inspect code}\n\n" <>
-      "Custom codes can be defined in the config file, using the code as the key " <>
-      "and the reason phrase as the value. For example:\n\n" <>
+    raise ArgumentError, "unknown status code #{inspect code}\n\n" <>
+      "Custom codes can be defined via the configuration of the :plug application, under " <>
+      "the :statuses key (which contains a map of status codes as keys and reason phrases " <>
+      "as values). For example:\n\n" <>
       "    config :plug, :statuses, %{451 => \"Unavailable For Legal Reasons\"}\n\n" <>
       "Doing this will allow the use of the atom :unavailable_for_legal_reasons " <>
       "when setting the status code. For example:\n\n" <>
       "    put_status(conn, :unavailable_for_legal_reasons)\n\n" <>
-      "After defining the config for custom statuses, recompile plug with:\n\n" <>
+      "After defining the config for custom statuses, be sure to recompile plug with:\n\n" <>
```

Comment:
The phrasing here feels a little casual compared to the rest of the error. What about:

> ... custom statuses, plug must be recompiled for the changes to take place using: